Sources/Sentry/Public/SentryGeo.h

```diff
@@ -0,0 +1,35 @@
+#import "SentryDefines.h"
+#import "SentrySerializable.h"
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+NS_SWIFT_NAME(Geo)
+@interface SentryGeo : NSObject <SentrySerializable, NSCopying>
+
+/**
+ * Optional: Human readable city name.
+ */
+@property (nullable, atomic, copy) NSString *city;
+
+/**
+ * Optional: Two-letter country code (ISO 3166-1 alpha-2).
+ */
+@property (nullable, atomic, copy) NSString *countryCode;
+
+/**
+ * Optional: Human readable region name or code.
+ */
+@property (nullable, atomic, copy) NSString *region;
```

Comment:
Where would this come from? Could you provide some examples in the headerdoc?

Reply:
I can only point you to @marandaneto's answer here: https://github.com/getsentry/sentry-cocoa/pull/2710#discussion_r1118840132

Reply:
This is automatically filled during ingestion when the `ipAddress` is set or inferred by the server.
One can manually set their own `geo` data tho, this is about that.
For example, a user might be located at X but using a VPN at Y.
Sometimes the inferred data isn't the preferred one.

Reply:
I guess what I'm really wondering is what is the expected format of data that would be in this field. Examples for countryCode would be `US`, `CZ` etc, I'm just not sure what would go in here. Is it just arbitrary and only meaningful to each separate app developer depending on their needs?

Reply:
@armcknight it's in the spec that I linked to in my [comment above](https://github.com/getsentry/sentry-cocoa/pull/2710#discussion_r1127816190).
https://develop.sentry.dev/sdk/event-payloads/types/#geo

Reply:
The spec lacks examples, @marandaneto. Could you please add some, as you added the spec https://github.com/getsentry/develop/pull/705?

Reply:
Yep, can do it, but lets not block this PR, example:

```
{
  "geo": {
    "country_code": "US",
    "city": "Ashburn",
    "region": "San Francisco"
  }
}
```

Reply:
@denrase, please add it to the code comments/docs in this repo. Thanks @marandaneto